src/meetingsession/MeetingSessionConfiguration.ts

```diff
@@ -81,7 +81,9 @@ export default class MeetingSessionConfiguration {
   reconnectLongBackOffMs = 5 * 1000;
 
   /**
-   * Feature flag to enable Chromium-based browsers
+   * Feature flag to enable Chromium-based browsers. This field is NOT
```

Comment:
Should we say that this configuration is deprecated and will be removed in future version and right now it will always be regarded as true since plan B ...

Reply:
Addressed